src/Components/OwnerInformation.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
Brand new component. Simple, clean, looks good. The heartbeat of the app could really lie in here (so long as users decide to write their stories).